code/modules/wiremod/shell/mech.dm

```diff
@@ -0,0 +1,203 @@
+/obj/vehicle/sealed/mecha/combat/durand/shell
+	desc = "An aging combat exosuit utilized by the Nanotrasen corporation, altered for autonomous control by circuits."
+	name = "\improper Circutix"
+
+/obj/vehicle/sealed/mecha/combat/durand/shell/Initialize()
+	. = ..()
+	AddComponent( \
+		/datum/component/shell, \
+		unremovable_circuit_components = list(new /obj/item/circuit_component/mech_movement, new /obj/item/circuit_component/mech_equipment), \
+		capacity = SHELL_CAPACITY_LARGE, \
+		shell_flags = SHELL_FLAG_ALLOW_FAILURE_ACTION \
+	)
+
+///Only exists to make porting a pain when any tg cooder realizes what a piece of shit this is
+/obj/vehicle/sealed/mecha/combat/durand/shell/proc/attack_target(atom/target, params)
```

Comment:
I hate this!
You should check if you can modify the procs that use attacker to accept null as well.